src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -243,10 +243,16 @@ private Run CreateSarifRun()
         {
             if (!string.IsNullOrWhiteSpace(originalUriBasePath))
             {
-                if (originalUriBasePath.StartsWith("/") &&
-                     platform == "Linux")
+                if (originalUriBasePath.StartsWith("/"))
                 {
-                    originalUriBasePath = "file://" + originalUriBasePath;
+                    if (platform == "Linux")
+                    {
+                        originalUriBasePath = "file://" + originalUriBasePath;
+                    }
+                    else
+                    {
+                        return null;
```

Comment:
> return null; [](http://example.com/codeflow?start=23&length=13)

This looks like an undesirable negative condition.

I'd either throw or I'd assert if you truly think it's dead code.

Reply:
changed to throw with useful user facing info.